### PR TITLE
Update Lambda Function code and Custom Resource.

### PR DESCRIPTION
**Pull request by groverlalit**
Issue: A change in the parameter only updates the lambda function and never invokes the updated lambda function.
Fix: Update CFN resources. Pass the parameters to the Custom Resource and process them in the Lambda function as an event. This allows the change in the parameters to update the IAM policy.

**(Override all values in parentheses)**

(Run `yamllint folder/template.yaml`, `cfn-lint -i E1019 E3002 E2520 -t folder/template.yaml`, and `aws cloudformation validate-template --template-body file://folder/template.yaml` before you open a PR)

(Do not include multiple changes in one PR. Open additional PRs instead.)

---

(describe your change here)

### CODE REVIEW

File: security/account-password-policy.yaml [hunk 1]
```diff
@@ -179,6 +179,14 @@ Resources:
     DependsOn: LambdaLogGroup
     Version: '1.0'
     Properties:
+      HardExpiry: !Ref HardExpiry
```
Comment (michaelwittig) [inline security/account-password-policy.yaml]: AllowUsersToChangePassword is missing

**Comment by michaelwittig** [comment]
thanks @groverlalit 
I found one small bug. Will merge as soon as this is fixed.

File: security/account-password-policy.yaml (hunk 1 above)
Comment (groverlalit) [reply security/account-password-policy.yaml]: Good catch!

File: security/account-password-policy.yaml [hunk 2]
```diff
@@ -141,24 +141,24 @@ Resources:
               const done = (err) => {
                 if (err) {
                   console.log(`Error: ${!JSON.stringify(err)}`);
-                  response.send(event, context, response.FAILED, {});
+                  response.send(event, context, response.FAILED, {}, context.log_stream_name);
```
Comment (michaelwittig) [inline security/account-password-policy.yaml]: can you revert the `, context.log_stream_name`
this was introduced in the 2nd commit and is the default

File: security/account-password-policy.yaml [hunk 3]
```diff
@@ -141,24 +141,24 @@ Resources:
...
                 } else {
-                  response.send(event, context, response.SUCCESS, {});
+                  response.send(event, context, response.SUCCESS, {}, context.log_stream_name);
```
Comment (michaelwittig) [inline security/account-password-policy.yaml]: can you revert the `, context.log_stream_name`
this was introduced in the 2nd commit and is the default

**Comment by michaelwittig** [comment]
Thanks @groverlalit

**Pull request merged**